packages/react-native-editor/__device-tests__/gutenberg-editor-audio.test.js

```diff
@@ -2,35 +2,35 @@
  * Internal dependencies
  */
 import { blockNames } from './pages/editor-page';
+import { waitForMediaLibrary } from './helpers/utils';
 import testData from './helpers/test-data';
-import { isAndroid } from './helpers/utils';
 
 describe( 'Gutenberg Editor Audio Block tests', () => {
 	it( 'should be able to add an audio block', async () => {
```

Comment:
With the consolidation of both test cases, I'm wondering if we should include here the description of the removed one (i.e. adding an audio file to the block), wdyt?

Reply:
https://github.com/WordPress/gutenberg/pull/39381#discussion_r829337753